backend/src/main/java/org/eclipse/tractusx/puris/backend/stock/logic/adapter/ItemStockSammMapper.java

```diff
@@ -75,35 +76,32 @@ private ItemStockSamm listToItemStockSamm(List<? extends ItemStock> itemStocks,
                     itemStock.getNonNullCustomerOrderId(), itemStock.getNonNullSupplierOrderId(),
                     itemStock.getNonNullCustomerOrderPositionId())));
```

Comment:
The lastUpdatedOnDateTime attribute is no longer a part of the "Position" member. Therefore, it is no longer required or sensible to filter for equal Dates here. I.e. please remove the Date attribute from the PositionsMappingHelper record an then also from line 74.